.changeset/healthy-llamas-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-llama": patch
+---
+
+feat(create-llama-pack): generate llama pack project from llama index
```

Comment:
@thucpn for changesets, `create-llama-pack` is not needed, as it's a `create-llama` patch file
```suggestion
feat: generate llama pack project from llama index
```